docs/src/main/sphinx/client/jdbc.rst

```diff
@@ -212,4 +212,6 @@ Name                                                              Description
                                                                   are treated as underscores. You can use this as a workaround for
                                                                   applications that do not escape schema or table names when passing them
                                                                   to ``DatabaseMetaData`` methods as schema or table name patterns.
+``ignoreBrokenCatalogs``                                          When enabled, the ``DatabaseMetaData`` methods to retrieve schemes, tables
```

Comment:
is it a left over?

Reply:
You are right, it is now a system session property, so it should not be listed here.

Reply:
why not remove it from here then?